### PR TITLE
[docs] added umask command to install YBA steps 

### PR DESCRIPTION
**Pull request by aishwarya24**
Slack request - https://yugabyte.slack.com/archives/CDQB8MEAU/p1708614538019359
Zendesk - https://yugabyte.zendesk.com/agent/tickets/8975

https://deploy-preview-21153--infallible-bardeen-164bc9.netlify.app/v2.18/yugabyte-platform/install-yugabyte-platform/install-software/installer/#quick-start

### CODE REVIEW

**Review by chidmuthu — CHANGES_REQUESTED** [review]
^

**Comment by aishwarya24** [comment]
> No, on versions <2.18.4.2 umask must be set to 022 before performing the install. Anything 2.18.4.2+ we don't have to worry about umask, YBA-installer handles it automatically.

Thought so. In that case, we don 't even need to mention it on the stable docs page as its addressing 2.20 series right? @chidmuthu

**Comment by chidmuthu** [comment]
> > No, on versions <2.18.4.2 umask must be set to 022 before performing the install. Anything 2.18.4.2+ we don't have to worry about umask, YBA-installer handles it automatically.
> 
> Thought so. In that case, we don 't even need to mention it on the stable docs page as its addressing 2.20 series right? @chidmuthu

Yes, but on 2.18 docs page we should call out on YBA-installer YBA install page if they are installing < 2.18.4.2 the umask will need to be set to 022 before performing the install

**Comment by aishwarya24** [comment]
@chidmuthu - Cuurently , we don't have an YBA installer upgrade page for 2.18 - https://docs.yugabyte.com/v2.18/yugabyte-platform/upgrade/. So ill create a similar one and update this PR. Thanks for the clarification.

**Comment by chidmuthu** [comment]
No this is not only for upgrades. It affects fresh install too. If they are doing upgrade they probably already set the umask (but it is not the worst to clarify on upgrades as umask should be set then as well)

**Review by chidmuthu — APPROVED** [review]

File: docs/content/v2.18/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ You can use YBA Installer only if you are about to perform a new install. Curren
   /bin/mv, /usr/bin/find, /opt/yugabyte/software/*/pgsql/bin/createdb, /opt/yugabyte/software/*/pgsql/bin/initdb
   ```
 
+- Note that if you are trying to install YBA installer less than version 2.18.4.2, you need to set umask for the root user to 022.
```
Comment (ddhodge) [inline docs/content/v2.18/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:62]:
```suggestion
- If you are installing a version earlier than v2.18.4.2, set umask for the root user to 022.
```

**Review by ddhodge — APPROVED** [review]

**Pull request merged**